ext/opentelemetry-ext-grpc/src/opentelemetry/ext/grpc/grpcext/_interceptor.py

```diff
@@ -164,6 +164,7 @@ def invoker(request_iterator, metadata):
         )
 
 
+# pylint: disable=abstract-method
```

Comment:
Was this causing lint issues? Shouldn't really be included as part of this PR.

Reply:
It was, fixed by #696 - will remove